srcCpp/perftest_cpp.h

```diff
@@ -18,8 +18,9 @@
 
 #ifdef RTI_WIN32
   #include <windows.h>
-#else
+#elif defined(RTI_VXWORKS)
```

Comment:
Correct me if I am wrong. Are you including <sys/time.h> if not RTI_WIN32 and RTI_VXWORKS?

Should it be?
```
#elif !defined(RTI_VXWORKS)
```